engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/aggregates.scala

```diff
@@ -237,6 +238,51 @@ object aggregates {
     }
   }
 
+  /*
+    Aggregator that holds two smaller aggregators and can add elements
+    to one of them. Adding Left(v) causes v to be added to first aggregator,
+    adding Right(v) causes v to be added to second aggregator. Result outputs
+    pair (result of first aggregator, result of second aggregator).
+   */
+  class EitherAggregator(val aggLeft: Aggregator, val aggRight: Aggregator) extends Aggregator {
```

Comment:
I wonder, wouldn't Map aggregate (with left/right branch names as keys) be enough?
What's more, the name is slightly misleading - as it receives Either, but stores tuple